src/packet/bwe/mod.rs

```diff
@@ -231,3 +240,149 @@ impl fmt::Display for BandwidthUsage {
         }
     }
 }
+
+/// Sliding window [`SeqNo`]s storage.
+///
+/// Only remembers the last [`SIZE`] packets added.
+#[derive(Debug)]
+struct HandledPacketsTracker<const SIZE: usize> {
```

Comment:
I'm not convinced that there's much value in the const generic here. `Box<[Option<u16>]>` seems sufficient to me.

Reply:
I like stack

Reply:
I don't think that really matters, tbh. Personally, I would be okay with simply hardcoding array size inside of struct since it's not being used anywhere anywhere else atm:

```rust
history: [u8; 64] // 64 bytes is a windows of 512 packets
```

Tell me how you'd like it to be.